README.Rmd

```diff
@@ -41,3 +41,13 @@ charts::demography |>
   ) |>
   charts::pyramid_chart(Typev = c("A", "B"), gendervar = "Sex", alpha_set = 0.7)
 ```
+
+## Figure 4 - Comorbidities (Grouped Bar Chart)
+
+```{r grouped_barchart, echo = FALSE}
+charts::grouped_barchart(
```

Comment:
remove reference to chart, e.g. charts::grouped_barchar to grouped_barchar and charts::comorbidities to comorbidities . In the first chunk add the line: devtools::load_all()

Reply:
Updated as suggested